packages/manager/src/features/CloudPulse/Alerts/AlertsListing/constants.ts

```diff
@@ -1,3 +1,8 @@
+import type {
+  AlertStatusType,
+  AlertStatusUpdateAction,
+} from '../../../../../../api-v4/lib/cloudpulse/types';
```

Comment:
```suggestion
} from '@linode/api-v4';
```

why it is importing from lib instead of src? import this from @linode/api-v4